src/boot/efi/meson.build

```diff
@@ -43,7 +43,7 @@ if not cc.has_header_symbol('efi.h', 'EFI_IMAGE_MACHINE_X64',
         subdir_done()
 endif
 
-objcopy = find_program('objcopy')
+objcopy = run_command(cc.cmd_array(), '-print-prog-name=objcopy', check: true).stdout().strip()
```

Comment:
@medhefgo do you remember why this was changed from find_program? now a prefixed `objcopy` can't be specified in a meson cross-file binaries section

i.e.
```meson
[binaries]
objcopy = 'aarch64-unknown-linux-gnu-objcopy'
```